src/lib/utils/healthbar.ts

```diff
@@ -0,0 +1,57 @@
+export type HealthType = "health" | "armor" | "shields";
+
+// Config
+export const CHUNK_SIZE = 25;
+
+// Colors
+export const COLORS: Record<HealthType, string> = {
+  health: "#ffffff",
+  armor: "#ee7c25",
+  shields: "#0be1ff",
+};
```

Comment:
I would personally prefer if these are moved to css variables which are defined via **s**css variables so that all colors are in one place, and are always defined in the css. It's fine if we keep that refactor for a later PR though